integration/tests/basic/test_batch_operations.py

```diff
@@ -0,0 +1,162 @@
+import random
+
+import pytest
+import requests
+from semantic_version import Version
+from web3 import Web3
+
+from integration.tests.basic.helpers import rpc_checks
+from utils.accounts import EthAccounts
+from utils.apiclient import JsonRPCSession
+from utils.web3client import NeonChainWeb3Client
+
+
+@pytest.mark.usefixtures("accounts", "web3_client")
+class TestBatchOperations:
+    accounts: EthAccounts
+    web3_client: NeonChainWeb3Client
+
+    def test_batch_operations_same_function(
+            self,
+            request: pytest.FixtureRequest,
+            common_contract,
+            json_rpc_client: JsonRPCSession,
+    ):
+        """
+        sends a batch operation request with 100 calls to the same function with different parameters
+        """
+        account = self.accounts[0]
+        evm_version: Version = self.web3_client.get_evm_version_number()
+        data_or_input_field_name = "data" if evm_version < Version("1.13.0") else "input"
```

Comment:
Can you please explain why this condition is here?

Reply:
the up-to-date documentation states that eth_call params expect field "input"
https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_call

but it turned out that with older evm versions, the field has to be "data"
https://github.com/ethereum/go-ethereum/issues/28608#issue-2010585064

Reply:
but it's not our evm version 
it's go ethereum
You check version or neon evm

You can check here only current behavior